templates/bpec-event-list.php

```diff
@@ -29,7 +31,11 @@
 <?php // list pagination
 
 if ( ! $my_events->have_posts() ) {
-	echo ( sprintf( __( 'There are no upcoming %s in your queue.', 'bp-events-calendar' ), $events_label_plural_lowercase ) );
+	if ( empty( $eventDisplay ) || 'past' !== $eventDisplay ) {
```

Comment:
@shamimmoeen sorry i did not understand the use of if else block, what is exactly we are doing here.

Reply:
If eventDisplay parameter is not set the url and no events is found then show this message 'There are no upcoming %s in your queue.' otherwise show this message 'There are no past %s in your queue.'

Reply:
@shamimmoeen  so first one for if there past events and no current events? Second one for if no events posted at all?